### DISCUSSION TITLE
Postgress Storage Issue

### DISCUSSION THREAD

**Discussion by duck-goes-quack** [vercel/community#2990] [Help]
### Summary

I have a simple database table of like 5 columns (one primary key) with 111 associated rows. When I create a database 30mb/250mb of the storage is set which is fine.

When doing serverless functions I have one which retrieves data to be used in the DOM on my website, this causes no storage issues. However, my PUT request serverless function is causing massive problems for me. It's literally a simple request to update ONE element in one individual row, changing it from "No" to "Yes". This somehow causes 30MB of increased storage each time, I was expecting like a few KB... so everytime I make a database I have to delete it completely since it goes over my plan.

I used a query to check how much data the table is holding and it said 70mb. I am 700mb/250mb right now somehow. I understand vercel does like 30mb by default so I should be max 100mb.

I tried asking chatgpt and it mentioned it could be to do with Logs & Temporary files and said to set a logging_collector parameter to off inside postgresql.conf - however I don't think I have this file, I wouldn't know where it is.

For reference I am using a github repos for my vercel hosted website. Please help :(
![Capture](https://github.com/vercel/community/assets/112182146/870c7556-63cc-405f-a58c-65a052d17b90)

### Steps to Reproduce

Can't replicate since it's an issue related to storage.

**Comment by Alimjan2013** [comment]
i think something wrong too, because when i check my usage, which is showing 560 MB / 256 MB. but i just store 2table , and one table just have 100 rows. 
<img width="734" alt="image" src="https://github.com/vercel/community/assets/65062804/b8fa9ea3-f0b9-4f67-aace-99d71c84da49">

**Reply by duck-goes-quack** [reply]
I see this is happening for a few people now, how do we contact support to let them know of the issue? Email is slow, are any of them on here?

**Reply by Alimjan2013** [reply]
i think this problem is generally exit, i believe vercel Storage team notice this problems. hope recovery soon.

**Comment by 10ccq** [comment]
Similar issue here. 
I didn't ever use or initialize my storage setting but when I check my usage, already 674 MB is used. I didn't even create any table or project.

![image](https://github.com/vercel/community/assets/136461693/c1b5dc2c-6da9-48b7-8b20-aec45a82c8f6)

**Reply by duck-goes-quack** [reply]
I see this is happening for a few people now, how do we contact support to let them know of the issue? Email is slow, are any of them on here?

**Comment by da-stoi** [comment]
Same issue here. I have under 20 rows across all tables
<img width="325" alt="image" src="https://github.com/vercel/community/assets/20031472/47320e72-10cd-4f1f-8b99-d96df3a93b75">

**Comment by amyegan** [comment]
Thanks for reporting this. We had a display bug related to storage size reporting which is fixed now. You should no longer see this issue, so please let me know if it is still happening for you 🙇
